com.unity.cinemachine/Editor/Upgrader/CinemachineUpgrader.cs

```diff
@@ -754,6 +771,8 @@ public void UpdateTimelineReference(CinemachineVirtualCameraBase upgraded)
                     {
                         var exposedRef = cmShot.VirtualCamera;
                         var vcam = exposedRef.Resolve(director);
+
+                        // This is problematic - what if there were already some null references in the project?
```

Comment:
That does not happen, because that cmShot was not collected in m_CmShotsToUpdate.